README.md

```diff
@@ -28,8 +28,9 @@ For pre-container setup, account prerequisites, and UI-based support, see our ar
 ## Installation Instructions
 
 1. Login with Docker credentials via `docker login`
-2. Pull latest image via `docker pull uptimecom/uptime-private-location:X.Y`
-3. Start the container via
+2. Retrieve latest stable image version [here](https://hub.docker.com/repository/docker/uptimecom/uptime-private-location/general).
+3. Pull latest image via `docker pull uptimecom/uptime-private-location:X.Y`
+4. Start the container via
```

Comment:
Looks good @thealexbrown , you may want to put #2 before #1, since logging into the dockerhub website is unrelated to `docker login` which is needed only for the `docker pull`?

Reply:
agree completely!